bundle/config/experimental.go

```diff
@@ -23,6 +23,22 @@ type Experimental struct {
 	// be removed in the future once we have a proper workaround like allowing IS_OWNER
 	// as a top-level permission in the DAB.
 	UseLegacyRunAs bool `json:"use_legacy_run_as,omitempty"`
+
+	// EnablePyDABs if enabled loads the 'databricks-pydabs' package.
+	//
+	// This allows to define bundle configuration using Python. For now, this option has to be used
+	// together with 'venv' configuration.
+	EnablePyDABs bool `json:"enable_pydabs,omitempty"`
```

Comment:
If there are additional things we need to make configurable, it would be better to have a `pydabs` section:

```yaml
pydabs:
  enable: true
  venv: <path>
```

Then you could punt on using the venv for artifact building etc.

Reply:
`venv` is a separate configuration tree because it should be possible to use it without enabling PyDABs

Speaking of `pydabs` configuration tree. I don't see how else we can possibly configure it except enabling, but overall, I agree that it's more futureproof. So I will change it, and we can think about the exact syntax when we remove from "experimental"